agent/03_agent_deployment.sh

```diff
@@ -13,6 +13,18 @@ source $SCRIPTDIR/agent/common.sh
 
 early_deploy_validation
 
+function add_host_dns_entries() {
+    ip_list=$1
+    num_ips=${#ip_list[@]}
+
+    for (( i=0; i<$num_ips; i++ ))
+    do
+        hostname=${FLEETING_HOSTNAME}${i}
+        sudo virsh net-update ${BAREMETAL_NETWORK_NAME} add dns-host  "<host ip='${ip_list[i]}'> <hostname>${hostname}</hostname> </host>"  --live --config
+    done
```

Comment:
As per the current imlementation, the same hostname could have an entry in the dhcp as well. What about removing such entry form the dhcp section, in order to have a cleaner/simpler environment?

Reply:
I've run into some intermittent issues when testing this since it requires restarting the network after that change. I don't want to make this fix now and introduce instability but will take it as a follow-on.